components/link-list/Component.jsx

```diff
@@ -80,26 +80,24 @@ export default function LinkList({ search }) {
       </div>
 
       <div
-        className="su-max-h-[1000px] su-h-0 su-overflow-hidden su-transition lg:su-h-auto"
+        className="su-max-h-1000 su-h-0 su-overflow-hidden su-transition lg:su-h-auto"
         id="link-drawer"
         data-role="link-drawer"
       >
         {linkItems[0] && (
-          <article className="su-border-solid su-border-b-[1px] su-border-b-black-20 dark:su-border-b-black-70 su-pb-[15px] su-mt-[23.65px] lg:su-pb-[36px]">
```

Comment:
I believe we no longer need su-border-solid to create solid a solid border. Also, if the border is 1px, you only need to write su-border